text/0000-no-more-extern-crate.md

```diff
@@ -70,29 +85,21 @@ For example, `cargo build`-ing a crate `my_crate` with a dependency on `rand`
 results in a call to rustc that looks something like
 `rustc --crate-name mycrate src/main.rs --extern rand=/path/to/librand.rlib ...`.
 
-When an external crate is specified this way,
-the crate will automatically brought into scope as if an
-`extern crate name_of_crate;`
-declaration had been added to the current crate root.
-This behavior won't occur when including a library using the `-l`
+When an external crate is specified this way, it will be automatically
+available to any module in the current crate through `use` statements or
+absolute paths (e.g. `::rand::random()`). It will _not_ be automatically
+imported at root level as happens with current `extern crate`.
```

Comment:
@petrochenkov The same for traits, they don't "enter" their naming scope inside of themselves, only modules do (which don't keep any of the outside scoping anyway).

Also apologies for not refreshing before sending that message, I assumed there was no reply.